package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "description": "Universal Redux Template",
   "scripts": {
-    "start": "NODE_PATH=./app nodemon app/server & node webpack.server.js & gulp css:watch",
-    "test": "NODE_ENV=test NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
+    "start": "cross-env NODE_PATH=./app concurrent --kill-others \"nodemon app/server\" \"node webpack.server.js\" \"npm run css:watch\"",
```

Comment:
The CSS job doesn't run since there's no npm script named `css:watch`. It should be `gulp css:watch`